content/docs/other-guides/multi-user-overview.md

```diff
@@ -151,6 +151,32 @@ The following resources are created as part of the profile creation:
 Namespaces, Profiles and Namespaces are sometimes used interchangably in the
 documentation.
 
+### Cloud API Authenticate with Profile
+
+**GCP**: User profile on GCP will be integrated with [GKE Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
+User pods will be authenticated automatically when talk to GCP APIs.
```

Comment:
"User pods will be authenticated automatically when talk to GCP APIs." -> 
"With workload identity, GCP credentials can be assigned to a pod by binding a Google Service Account to the Kubernetes Service Account used by the pod."